File: ProAgil/ProAgil.Dominio/Palestrante.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ProAgil.Dominio
{
    public class Palestrante
    {
        public int Id { get; set; }

        public string Nome { get; set; }

        public string MiniCurriculo { get; set; }

        public string ImagemURL { get; set; }

        public string Telefone { get; set; }

        public string Email { get; set; }

        public List<RedeSocial> RedesSocials { get; set; }

        public List<PalestranteEvento> PalestrantesEventos { get; set; }
    }
}

File: ProAgilApp/src/app/_models/Palestrante.ts
import { RedeSocial } from "./RedeSocial";
//import { Palestrante } from "./Palestrante";

export interface Palestrante {
    id: number; 
    nome: string; 
    miniCurriculo: string;
    imagemURL: string;
    telefone: string;
    email: string;
    redesSociais: RedeSocial[];
    //palestrantesEventos: Palestrante[];
}
